File: library/src/main/java/com/hy/wanandroid/library/util/ScreenUtils.java
package com.hy.wanandroid.library.util;

import android.content.Context;
import android.content.res.Resources;

/**
 * Created by huangyong on 2017/9/25.
 * 手机屏幕相关工具类
 */

public final class ScreenUtils {
    private ScreenUtils() {
    }

    /**
     * 获取屏幕宽度
     */
    public static int getScreenWidth(Context context) {
        Resources resources = context.getResources();
        return resources.getDisplayMetrics().widthPixels;
    }
}

File: library/src/main/java/com/hy/wanandroid/library/util/ScreenUtils.kt
package com.hy.wanandroid.library.util

import android.content.Context

/**
 * Created by huangyong on 2017/9/25.
 * 手机屏幕相关工具类
 */
object ScreenUtils {
    /**
     * 获取屏幕宽度
     */
    fun getScreenWidth(context: Context): Int {
        val resources = context.resources
        return resources.displayMetrics.widthPixels
    }
}